bundle/Resources/views/default/content/text_linked.html.twig

```diff
@@ -2,7 +2,7 @@
 {# location \Netgen\IbexaSiteApi\API\Values\Location #}
 
 {% if location is defined %}
-    <a href="{{ ibexa_path(location) }}">{{ content.name }}</a>
+    <a href="{{ location.url }}">{{ content.name }}</a>
```

Comment:
This is quite a change in nomenclature here. We all know what `path` and `url` are in Symfony, and changing this here could cause confusion. I would rather have two properties `location.path` and `location.url` and then further refine `location.path` with `location.path.relative`, `location.path.absolute` and `location.path.network_relative`.

Reply:
I second this, it would be better to be in sync with Symfony nomenclature.

Reply:
Now I realize that `location.path` is taken by Ibexa location already, but I would be fine overriding it in Site API. If you need access to it, you would use `location.innerLocation.path` or even using a renamed property `location.pathArray`

Reply:
Good points, will update 👍

Reply:
Changed in https://github.com/netgen/ibexa-site-api/pull/11/commits/59eedfdf273936da2cdc10397bc92ca1c13ae898 and updated the description, is this what you had in mind?

Reply:
Yep, looks good! 👍

Reply:
Great, then the PR on the `netgen/site-bundle` for `Location::$path` => `Location::$pathArray` change follows.